starwars/src/App.js

```diff
@@ -33,6 +34,11 @@ class App extends Component {
     return (
       <div className="App">
         <h1 className="Header">React Wars</h1>
+        {this.state.starwarsChars[0] ? (
```

Comment:
You do not need this because if you have state with the [ ] it will map over it but not see it as undefined because map can iterate over an empty array. It just has nothing appear. If our state was state: { starwarsChars: undefined } then we would need the ternary.